frontend.rst

```diff
@@ -33,6 +33,13 @@ to solve the most common Webpack use cases.
 
 .. _encore-toc:
 
+Technical Requirements
+----------------------
+
+* Node.js on your development machine(s)
+* Yarn or npm on your development machine(s)
```

Comment:
Maybe add links to those binary installation process?

Reply:
OK, done :-)
npm is part of Node.js, right? - So no need for a link?!